### PR TITLE
Made messages look more stylish, and (re)added stars to unread icons.

### LINKED ISSUE #372: [SIS] Unread messages have no labeling
previously unread messages were marked as such with a star using css on the 'unread' class

### PR DESCRIPTION
**Pull request by mstratford**
Fixes #372 

Also makes the popups more pretty and adds a few tooltips 👍 

**Testing Completed**

I used my own local copies of sis.css and sis.messages.js to override on live SIS, so should be ready to go 👍

### CODE REVIEW

**Comment by mstratford** [comment]
http://imgur.com/a/W7xH4 for a preview :)

**Comment by Brookke** [comment]
hmm i like although not sure starts are helpful as they usually indicate a favourite in most UI's ( I know they previously were starts though)  a circle might be more appropriate.

**Comment by mstratford** [comment]
I'll eye-drop the colour of the iOS/MacOS mail unread dot and copy it 👍

**Comment by Brookke** [comment]
haha doesn't have to be that exact ;)

**Comment by AJDurant** [comment]
You could potentially make the preview text better, sometimes on long
messages it's a bit weird. Something like a fixed size trim with an
ellipsis?

**Comment by mstratford** [comment]
Was thinking about that, i’ll get on it now :)

**Comment by mstratford** [comment]
OK, I've done all of that :) Ready for merge!

File: src/Public/css/sis.css [hunk 1]
```diff
@@ -45,9 +45,22 @@
...
+.td-msgitem.unread td:first-child {
+  visibility: visible;
+}
+.td-msgitem td:nth-child(2) {
+  width: 15px;
+  padding-right: 35px;
+}
+.td-msgitem .unread-dot {
+  border-radius: 100%;
+  height:14px;
```
Comment (Brookke) [inline src/Public/css/sis.css]: might be worth using em here then they scale with the font size

Comment (mstratford) [inline src/Public/css/sis.css]: Font size doesn't change in show planner though by the looks of it?

Comment (Brookke) [inline src/Public/css/sis.css]: True, but people might be running their browsers with a different font size.

Comment (mstratford) [inline src/Public/css/sis.css]: done :)

File: src/Public/js/sis.messages.js [hunk 2]
```diff
@@ -86,7 +86,14 @@ var Messages = function () {
 
                 locationTd.innerHTML = icon_location;
                 unreadTd.innerHTML = icon_unread;
-                titleTd.innerHTML = data[i]['title'];
+
+                var longTitle = data[i]['title'];
+                var trimmedTitle = longTitle.substring(0, 120);
+                if (longTitle != trimmedTitle) {
+                    trimmedTitle = trimmedTitle + '... <a>Read more</a>';
```
Comment (AJDurant) [inline src/Public/js/sis.messages.js]: If someone clicks on read more, does it still work, or does it's anchor tag with no destination override to do nothing?

Comment (mstratford) [inline src/Public/js/sis.messages.js]: Just tried it, it doesn't have any effect, so still opens the popup.

Comment (AJDurant) [inline src/Public/js/sis.messages.js]: Excellent

**Pull request merged**